substrate/primitives/weights/src/weight_v2.rs

```diff
@@ -401,14 +401,14 @@ where
 	}
 }
 
-#[cfg(any(test, feature = "std", feature = "runtime-benchmarks"))]
```

Comment:
Better not to add some conditional code for runtime-benchmark here IMO.
This code was never generated, better not to start generating it.

Reply:
Yea feel free to delete. I normally generate [helpers](https://github.com/paritytech/polkadot-sdk/blob/7ecf3f757a5d6f622309cea7f788e8a547a5dce8/substrate/frame/message-queue/src/mock_helpers.rs#L34) on the fly if i need to.